### PR TITLE
Close ZotHacks 2024 Hacker and Mentor Applications

### PR DESCRIPTION
**Pull request by waalbert**
- Switches from the Apply to Apps Closed button on the Landing with a grey hover effect
- Disables Get Involved section buttons
- Removes Apply button from Footer
- Redirects `/apply` and `/mentor` routes to the home page

### CODE REVIEW

File: apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.tsx [hunk 1]
```diff
@@ -12,14 +12,19 @@ const GetInvolved = () => {
 		<p>Otherwise, if you have some experience under your belt,</p>
 	);
 	const applyLink = (
-		<Button href="/apply" type="button" className={styles.applyButton}>
-			Apply as a Hacker
+		<Button href="/apply" type="button" className={styles.applyButton} disabled>
```
Comment (Bl20052005) [inline apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.tsx:15]: href still here (already mentioned but still noting)

also, should the button still have a hover state even when apps are closed?

File: apps/site/src/app/(home)/sections/Landing/ApplyButton.module.scss [hunk 2]
```diff
@@ -28,11 +29,11 @@
 
 svg:hover {
```
Comment (Bl20052005) [inline apps/site/src/app/(home)/sections/Landing/ApplyButton.module.scss:30]: repeating the above but should svgs change if the apps are closed?

**Review by Bl20052005 — DISMISSED** [review]

File: apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.tsx (hunk 1 above)
Comment (IanWearsHat) [reply apps/site/src/app/(home)/sections/GetInvolved/GetInvolved.tsx:15]: The `/apply` route redirects to home anyways, so I think it's fine.

The hover state is still there for style. It doesn't look like the hover state for when the button is active. It gives greyed out feedback, which I think looks better than a simple disabled button.

Same thing for the svgs, it'll just change to the greyed out disabled button.

**Review by IanWearsHat — APPROVED** [review]

**Comment by IanWearsHat** [comment]
I also removed the apply link from sanity FAQ for the question, "Do I apply as a team or individual?"

**Pull request merged**